DOCS/man/vo.rst

```diff
@@ -300,8 +300,16 @@ Available video output drivers are:
     as ``auto-safe``. It can still work in some circumstances without ``--hwdec`` due to
     mpv's internal conversion filters, but this is not recommended as it's a needless
     extra step. Correct output depends on support from your GPU, drivers, and compositor.
-    Weston and wlroots-based compositors like Sway and Intel GPUs are known to generally
-    work.
+    This requires the compositor and mpv to support ``color-management-v1`` to
```

Comment:
Technically we're using `xx-color-management-v4`.

Reply:
I could change that but it would need to be updated then I suppose